### PR TITLE
Feature/governance

### PR DESCRIPTION
**Pull request by iStrike7**
Added the collective pallet, phragmen elections pallet and created a sudo-origin pallet.

No breaking API changes.

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -571,6 +573,62 @@ impl orml_tokens::Trait for Runtime {
...
+	type ModuleId = ElectionsPhragmenModuleId;
+	type Currency = orml_tokens::CurrencyAdapter<Runtime, MGATokenID>;
+	type ChangeMembers = Council;
+	// NOTE: this implies that council's genesis members cannot be set directly and must come from
+	// this module.
+	type InitializeMembers = Council;
+	type CurrencyToVote = CurrencyToVoteHandler;
+	type CandidacyBond = CandidacyBond;
+	type VotingBond = VotingBond;
+	type LoserCandidate = (); // TODO Treasury
```
Comment (mateuszaaa) [inline runtime/src/lib.rs]: can we use Treasury right away?

File: runtime/src/weights/pallet_collective.rs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+//! Default weights for the Collective Pallet
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0-rc6
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::weights::{Weight, constants::RocksDbWeight as DbWeight};
+
+pub struct WeightInfo;
+impl pallet_collective::WeightInfo for WeightInfo {
```
Comment (mateuszaaa) [inline runtime/src/weights/pallet_collective.rs:25]: i remember you mentioned that we will have to figure out proper numbers - maybe we should create task for that ?

File: pallets/sudo-origin/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Sudo Module
```
Comment (mateuszaaa) [inline pallets/sudo-origin/README.md:1]: that one seems to be outdated - maybe consider removing it for now ?

File: pallets/sudo-origin/src/lib.rs [hunk 4]
```diff
@@ -0,0 +1,251 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2020 Parity Technologies (UK) Ltd.
```
Comment (mateuszaaa) [inline pallets/sudo-origin/src/lib.rs:3]: Mangata license ?

File: runtime/src/lib.rs (hunk 1 above)
Comment (mateuszaaa) [reply runtime/src/lib.rs]: ah, its done already :+1:

File: runtime/src/weights/pallet_collective.rs (hunk 2 above)
Comment (iStrike7) [reply runtime/src/weights/pallet_collective.rs:25]: Created trello task for it: https://trello.com/c/lozlF5QH/263-remove-members-in-pallet-phragmen-elections-require-maximumblockweight

File: pallets/sudo-origin/README.md (hunk 3 above)
Comment (iStrike7) [reply pallets/sudo-origin/README.md]: Edited it.

File: pallets/sudo-origin/src/lib.rs (hunk 4 above)
Comment (iStrike7) [reply pallets/sudo-origin/src/lib.rs]: Removed Substrate licensing.

**Review by mateuszaaa — APPROVED** [review]
LGTM

**Review by gleb-urvanov — APPROVED** [review]
LGTM,
However we will need a governance API documentation, explanation of the election process and the default values for term duration, council seats number etc

**Pull request merged**